### PR TITLE
feat: add vue-strong multi attribute element rule plus fixes

### PR DESCRIPTION
**Pull request by David-Pena**

### CODE REVIEW

File: src/rules/rules.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export const RULES = {
     'simpleComputed',
     'templateSimpleExpression',
     'fullWordComponentName',
+    'multiAttributeElements',
```
Comment (rrd108) [inline src/rules/rules.ts:15]: Can we keep alphabetic order here?

File: src/rulesReport.ts [hunk 2]
```diff
@@ -68,13 +70,15 @@ export const reportRules = (groupBy: GroupBy) => {
   processOffenses(reportSimpleComputed)
   processOffenses(reportComponentFiles)
```
Comment (rrd108) [inline src/rulesReport.ts:71]: Can we keep alphabetic order here?

**Review by rrd108 — CHANGES_REQUESTED** [review]

File: src/rules/rules.ts (hunk 1 above)
Comment (David-Pena) [reply src/rules/rules.ts:15]: on it

File: src/rulesReport.ts (hunk 2 above)
Comment (David-Pena) [reply src/rulesReport.ts:71]: you mean per section? like this:

processOffenses(reportComponentFilenameCasing)
processOffenses(reportComponentFiles)
processOffenses(reportDirectiveShorthands)
processOffenses(reportFullWordComponentName)
processOffenses(reportMultiAttributeElements)
processOffenses(reportPropNameCasing)
processOffenses(reportQuotedAttributeValues)
processOffenses(reportSelfClosingComponents)
processOffenses(reportSimpleComputed)
processOffenses(reportTemplateSimpleExpression)

Comment (David-Pena) [reply src/rulesReport.ts:65]: fix in last commit

**Review by rrd108 — APPROVED** [review]

**Pull request merged**